src/utils/Firebase/firebase.js

```diff
@@ -120,6 +125,9 @@ class Firebase {
                   let lessonCopy = JSON.parse(JSON.stringify(customLesson));
                   delete lessonCopy.dueDates;
                   lessonCopy.dueDate = dueDate;
+                  lessonCopy.isCompleted = deploymentAccountDoc.get(
+                    "customLessons"
```

Comment:
if you use this str several times in this file make it a const at the top and use that instead

Reply:
wait which string?

Reply:
`"customLessons"`